floss/main.py

```diff
@@ -349,19 +357,21 @@ def select_functions(vw, asked_functions: Optional[List[int]]) -> Set[int]:
     return asked_functions_
 
 
-def is_supported_file_type(sample_file_path: Path):
+def get_file_type(sample_file_path: Path):
     """
-    Return if FLOSS supports the input file type, based on header bytes
+    Returns input file type, based on header bytes
     :param sample_file_path:
-    :return: True if file type is supported, False otherwise
+    :return: file type if it is supported, raise Error Otherwise
     """
     with sample_file_path.open("rb") as f:
-        magic = f.read(2)
+        magic = f.read(4)
 
-    if magic in SUPPORTED_FILE_MAGIC:
-        return True
+    if magic == SUPPORTED_FILE_MAGIC_ELF:
+        return SUPPORTED_FILE_MAGIC_ELF
+    elif magic[:2] == SUPPORTED_FILE_MAGIC_PE:
+        return SUPPORTED_FILE_MAGIC_PE
     else:
-        return False
+        return None
```

Comment:
In the docstring is written:` file type if it is supported, raise Error Otherwise` but you are returning None here.

Is this correct?

Reply:
Oh i will update that

Also will rebase the code to master as well.